src/devtools/views/Components/types.js

```diff
@@ -65,6 +65,11 @@ export type InspectedElement = {|
   source: Object | null,
 |};
 
+export type InspectedElementResponse = {|
+  id: number,
+  inspectedElement: InspectedElement | null,
```

Comment:
I'm not sure if this is the most graceful/intuitive way to model this. Essentially we need to distinguish between three possible states:
* No element was found for the specified ID (currently returns `null`).
* An element was found, and here are its props (currently returns a `InspectedElementResponse` with an `InspectedElement` inside).
* The element has not been rendered/committed since the last time it was inspected (currently returns a `InspectedElementResponse` with a null value inside).

Maybe this optimization is premature. I think it might help with cases like the Destiny Item Manager, although I don't think it's sufficient to fix those cases (since hovering over a lot of elements may still cause us to request a lot of inspections initially). So maybe I should just revert this particular change and follow up via #195

Reply:
I just reverted this change for now (0aa7d2f). We can re-visit it with an eye to perf for cases like D.I.M.